packages/chat/components/chat/__stories__/chat.mdx

```diff
@@ -175,6 +183,39 @@ There are three ways to implement Chat: Add an API and directly auto-parse raw L
 </clabs-chat>
 ```
 
+### Usage in React:
+<a id="react-implementation"></a>
+For React, one extra step is needed:
+```js
+//ChatComponent.js
+import {createComponent} from '@lit/react';
+import React from 'react';
+import CLABSChat from '@carbon-labs/ai-chat/es/components/chat/chat.js';
+
+export const Chat = createComponent({
+  tagName: 'clabs-chat',
+  elementClass: CLABSChat,
+  react: React,
+  events: {
+  },
+});
+```
```

Comment:
I think it would be nice if we included the React wrappers as part of the chat package (just like how `feedback` has the `react` folder) so folks don't have to create this on their end themselves and can just import it, but we can merge this in for now